### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (83)

This is the login page of my E-commerce website. Please review it!

### CODE REVIEW

**Review by Muzammil-khan710 — APPROVED** [review]
All good. Good to merge !

File: css/navbar.css [hunk 1]
```diff
@@ -54,8 +25,32 @@
...
+}
+
+.right-nav {
+    height: 0rem;
+    align-content: center;
+}
+
+.btn {
+    margin: 10px;
+    padding: 0.7rem;
```
Comment (zoyatanweer) [inline css/navbar.css:45]: convert this into pixels

File: css/root.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    font-size: 1rem;
+}
+
+.justify-align {
+    justify-content: center;
+    align-items: center;
+}
+
+.title {
+    margin: 0.625rem 0rem 1.25rem 0.625rem;
```
Comment (zoyatanweer) [inline css/root.css:23]: these values can be written in pixels.
0.625rem is very small, I would suggest you to convert them in pixels

File: css/signup-login.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.form-container {
+    width: 25%;
+    height: 65%;
+}
+
+.input-box {
+    width: 100%;
+    margin: 0.625rem 0rem;
```
Comment (zoyatanweer) [inline css/signup-login.css:19]: here too write in pixels

**Review by Shubhamkr2610 — APPROVED** [review]
All goods to me. You are good to go

File: style.css [hunk 4]
```diff
@@ -1,66 +1,72 @@
...
     border-radius: 40%;
     background-color: var(--box-color);
     position: relative;
 }
-   
-.text-box{
+
+.text-box {
     position: absolute;
     font-size: 20px;
```
Comment (zoyatanweer) [inline style.css]: this value can be converted into rem

File: style.css [hunk 5]
```diff
@@ -1,66 +1,72 @@
...
+
+.img-category {
     display: flex;
     flex-direction: column;
     align-items: center;
 }
-.hero-image{
+
+.hero-image {
     margin-bottom: 20px;
```
Comment (zoyatanweer) [inline style.css]: write in rem here too

File: style.css [hunk 6]
```diff
@@ -1,66 +1,72 @@
...
 }
-.cards-category{
+
+.cards-category {
     display: flex;
     margin: 20px;
```
Comment (zoyatanweer) [inline style.css]: make changes here too for rem

File: style.css [hunk 7]
```diff
@@ -1,66 +1,72 @@
...
+    margin-top: 20px;
+}
+
+.card-text {
+    color: var(--accent-color);
+    margin: 10px;
+}
+
+.res-image {
+    height: 450px;
```
Comment (zoyatanweer) [inline style.css]: write this in rem

**Review by zoyatanweer — APPROVED** [review]
do the suggested changes, UI looks easy on the eyes. Code is clean and easy to understand. Good to merge with suggested changes.

**Pull request merged**